### PR TITLE
Single item select

### PR DESCRIPTION
**Pull request by pstjean**

### CODE REVIEW

File: projects/ngx-prx-styleguide/src/lib/select/select.component.css [hunk 1]
```diff
@@ -11,4 +11,12 @@
   flex-basis: 215px;
   -ms-flex-preferred-size: 215px;
   -webkit-flex-basis: 215px;
+}
+
+:host /deep/ .max-select-one .ng-value-icon {
```
Comment (cavis) [inline projects/ngx-prx-styleguide/src/lib/select/select.component.css]: Fancy CSS!

File: projects/ngx-prx-styleguide/src/lib/select/select.component.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { Component, Input, Output, EventEmitter, OnChanges, SimpleChanges } from '@angular/core';
-import { NgOption, NgSelectConfig } from '@ng-select/ng-select'
+import { Component, Input, Output, EventEmitter, OnChanges, SimpleChanges, ViewChild } from '@angular/core';
```
Comment (cavis) [inline projects/ngx-prx-styleguide/src/lib/select/select.component.ts]: What's the change in imports for?

Comment (cavis) [reply projects/ngx-prx-styleguide/src/lib/select/select.component.ts]: (Maybe don't need that `ViewChild`)

Comment (pstjean) [reply projects/ngx-prx-styleguide/src/lib/select/select.component.ts]: Related to my failed attempt to use templates for this. Removed.

**Pull request merged**